blade/procData/Scripts/simulatedResults/testGenerator.py

```diff
@@ -135,7 +135,23 @@ def generate_testLog(orgName, startTime, endTime, logAmount):
 
     logFile.close()
 
+def grabEmployeeSample(jsonFileName, employeeAmount):
+    names=[]
+    employeeCount = 0
+    jsonFile = open(jsonFileName, 'r')
+    namePool = json.load(jsonFile)
+    while employeeCount <= employeeAmount:
```

Comment:
good use of while, you could also do a `while True` and then break when the condition becomes true